doc/source/rllib/rllib-algorithms.rst

```diff
@@ -7,236 +7,176 @@
-`[implementation] <https://github.com/ray-project/ray/blob/master/rllib/algorithms/bc/bc.py>`__
-
-Our behavioral cloning implementation is directly derived from our `MARWIL`_ implementation,
-with the only difference being the ``beta`` parameter force-set to 0.0. This makes
-BC try to match the behavior policy, which generated the offline data, disregarding any resulting rewards.
-BC requires the `offline datasets API <rllib-offline.html>`__ to be used.
-
-Tuned examples: `CartPole-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/bc/cartpole-bc.yaml>`__
-
-**BC-specific configs** (see also `common configs <rllib-training.html#common-parameters>`__):
-
-.. autoclass:: ray.rllib.algorithms.bc.bc.BCConfig
-   :members: training
+Overview
+~~~~~~~~
+
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| **Algorithm**                                                               | **Single- and Multi-agent**  | **Multi-GPU (multi-node)**         | **Action Spaces**              |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| **On-Policy**                                                                                                                                                                    |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| :ref:`PPO (Proximal Policy Optimization) <ppo>`                             | |single_agent| |multi_agent| | |multi_gpu| |multi_node_multi_gpu| | |cont_actions| |discr_actions| |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| **Off-Policy**                                                                                                                                                                   |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| :ref:`DQN/Rainbow (Deep Q Networks) <dqn>`                                  | |single_agent| |multi_agent| | |multi_gpu| |multi_node_multi_gpu| |                |discr_actions| |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| :ref:`SAC (Soft Actor Critic) <sac>`                                        | |single_agent| |multi_agent| | |multi_gpu| |multi_node_multi_gpu| | |cont_actions|                 |
```

Comment:
That's actually not correct. Mulit-learner settings are not available for SAC  - due to the multiple optimizers per learner.

Reply:
fixed